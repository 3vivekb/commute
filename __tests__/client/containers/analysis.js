/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStores} from '../../test-utils/mock-data.js'

import Analysis from '../../../client/containers/analysis'

const mockStore = makeMockStore(mockStores.init)

describe('Container > Analysis', () => {
  it('Analysis View loads (base case)', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <Analysis />
      </Provider>
    )
    expect(mountToJson(tree.find(Analysis))).toMatchSnapshot()
  })

  it('Analysis View loads (commuter group or site has changed)', () => {
    throw new Error('unimplemented')
  })

  it('Analysis View loads (analysis is being calculated)', () => {
    throw new Error('unimplemented')
  })

  it('Update Analysis', () => {
    throw new Error('unimplemented')
  })

  it('Delete analysis', () => {
    throw new Error('unimplemented')
  })
})
