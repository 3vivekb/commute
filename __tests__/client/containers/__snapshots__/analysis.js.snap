exports[`Container > Analysis Analysis View loads (base case) 1`] = `
<Connect(Analysis)>
  <Analysis
    dispatch={[Function]}>
    <h1>
      Analysis
    </h1>
  </Analysis>
</Connect(Analysis)>
`;
